### PR TITLE
Login

### PR DESCRIPTION
**Pull request by nrrnie**

### CODE REVIEW

File: utils/API.py [hunk 1]
```diff
@@ -16,7 +16,13 @@ def find_user_by_barcode(self, barcode: int) -> dict:
         link = self.host + '/users/user_by_barcode/%d' % barcode
         response = API.safe_request(link)
         return response if response['status'] == 'error' else response['response']
-
+        
+    def login(self, barcode: int, password: str) -> dict:
+        link = self.host + '/auth/login'
+        data = dict(barcode=barcode, password=password)
+        response = API.safe_request(link, data=data)
+        return response if response['status'] == 'error else response['response']
```
Comment (Mercurial5) [inline utils/API.py:24]: didn't close a single quote after error

File: aituNetwork/auth/routes.py [hunk 2]
```diff
@@ -29,6 +29,23 @@ def sign_up():
 def sign_in():
     if request.method == 'GET':
         return render_template('sign_in.html')
+        
+@auth.route('/login', methods=['GET', 'POST'])
```
Comment (Mercurial5) [inline aituNetwork/auth/routes.py:33]: API will return data only for POST requests, so you need to remove 'GET' method

**Review by Mercurial5 — CHANGES_REQUESTED** [review]

File: aituNetwork/auth/routes.py (hunk 2 above)
Comment (nrrnie) [reply aituNetwork/auth/routes.py:33]: true

**Review by nrrnie — COMMENTED** [review]
cool

**Review by Mercurial5 — APPROVED** [review]

**Pull request merged**